lib/ts/recipe/thirdpartyemailpassword/emaildelivery/services/smtp/index.ts

```diff
@@ -0,0 +1,52 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+import {
+    ServiceInterface,
+    GetContentResult,
+    TypeInputSendRawEmail,
+    TypeInput as SMTPTypeInput,
+    getEmailServiceImplementation,
+} from "../../../../../ingredients/emaildelivery/services/smtp";
+import { Transporter } from "nodemailer";
+import { TypeThirdPartyEmailPasswordEmailDeliveryInput } from "../../../types";
+import getPasswordResetEmailContent from "../../../../emailpassword/emaildelivery/services/smtp/passwordReset";
+import getEmailVerifyEmailContent from "../../../../emailverification/emaildelivery/services/smtp/emailVerify";
+
```

Comment:
This is not the same structure as the other services (a class that implements the service interface)